config/musonza_chat.php

```diff
@@ -23,13 +16,41 @@
     'sent_message_event' => 'Musonza\Chat\Eventing\MessageWasSent',
 
     /*
-     * Automatically convert conversations with more than two users to public
+     * Specify the fields that you want to return each time for the sender.
+     * If not set or empty, all the columns for the sender will be returned
+     *
+     * However, if using multiple Models it's recommended to add getParticipantDetails to each
+     * Model you want to control fields output.
      */
-    'make_three_or_more_users_public' => true,
 
     /*
      * Specify the fields that you want to return each time for the sender.
      * If not set or empty, all the columns for the sender will be returned
      */
     'sender_fields_whitelist' => [],
+
+    /*
+     * Whether to load the package routes file in your application.
+     */
+    'should_load_routes' => true,
+
+    /*
+     * Routes configuration
+     */
+    'routes' => [
+        'enabled'     => false,
+        'path_prefix' => 'chat',
+        'middleware'  => ['web'],
+    ],
+
+    /*
+     * Default values for pagination
+     */
+    'pagination' => [
```

Comment:
You could put a really big number? At some point loading all the messages will have negative impact as the number of messages grow. Any issues with pagination? Otherwise i can add a no pagination config value

Reply:
I have done that for now. but yeah then I need to do some ajax call when the user scrolls to fetch new messages. If I should do it the right way